extensions-core/parquet-extensions/src/main/java/org/apache/druid/data/input/parquet/ParquetExtensionsModule.java

```diff
@@ -46,14 +60,42 @@
             .registerSubtypes(
                 new NamedType(ParquetAvroHadoopInputRowParser.class, PARQUET_AVRO_INPUT_PARSER_TYPE),
                 new NamedType(ParquetHadoopInputRowParser.class, PARQUET_SIMPLE_INPUT_PARSER_TYPE),
-                new NamedType(ParquetParseSpec.class, PARQUET_SIMPLE_INPUT_PARSER_TYPE)
+                new NamedType(ParquetParseSpec.class, PARQUET_SIMPLE_INPUT_PARSER_TYPE),
+                new NamedType(DruidNativeParquetInputFormat.class, PARQUET_SIMPLE_PARSE_SPEC_TYPE)
             )
     );
   }
 
   @Override
   public void configure(Binder binder)
   {
+    final Configuration conf = new Configuration();
+
+    // Set explicit CL. Otherwise it'll try to use thread context CL, which may not have all of our dependencies.
```

Comment:
This is copied from the HDFS module to initialize Hadoopy things... Not sure of a good way to share this because it requires Hadoop libraries that core Druid doesn't have...

Reply:
Agree it would be good to mention this code is copied from HdfsStorageDruidModule, and why. (And a similar comment in that file.)

That way, if someone modifies it in the future to fix a problem, they'll hopefully remember to fix both places.